scripts/util.ts

```diff
@@ -57,3 +57,25 @@ export const awaitBlockProduction = async (wsUrl: string) => {
 		logWithDate(chalk.blue('Polkadot-js successfully disconnected'));
 	});
 };
+
+export const awaitEpochChange = async (api: ApiPromise) => {
```

Comment:
I see the ending logs reference HRMP channels being opened. Will we likely exclusively use this for waiting to open HRMP channels?

Reply:
Cool, I understand that part of it. My question was more about semantics given the functions name and the mentioning of HRMP channels in its logs. Not a big deal right now and can always be revisited :)